src/CMakeLists.txt

```diff
@@ -130,10 +103,6 @@ set (NN_SOURCES
     utils/sleep.c
     utils/thread.h
     utils/thread.c
-    utils/thread_posix.h
```

Comment:
What I was saying, or suggesting, was that I'd revert this portion of the change.  I think the old stuff worked, didn't it?  (Even when applied to cmake?)

My goal is to have the changeset required to enable cmake be the smallest change we can easily do), since and then refine it going forward.